js/year_carte.js

```diff
@@ -1414,9 +1414,11 @@ $(function () {
     });
 
     // Switch the year from the dropdown
-    $('#year-select-AllPrenom').change(function () {
+    $('#autocomplete-input').change(function(){
+      setTimeout(function () {
+        var val = $('#autocomplete-input').val();
+        console.log('Updating chart for: ' + val);
```

Comment:
I used this to noticed the value was wrong. I let it for you to check it in your console. (Right click -> Inspect -> Console)